brave/brave-ios
# Fix #6680: Use Solana Web3 library script from BraveCore

## Summary of Changes
- Removed checked-in SolanaWeb3.js library file 
- Use the SolanaWeb3 script from core now that fix is applied there: https://github.com/brave/brave-browser/issues/27325

This pull request fixes #6680

## Submitter Checklist:


## Test Plan:
- Verify Solana dapps are still working as expected.
- Verify prototype pollution fix still applies as expected
> 1. Open dev console on any website on page load
> 2. Input `Object.freeze = ()=>{}`
> 3. Input `await solana.connect()` and approve the connect request
> 4. We should see publicKey has value without any console errors
> Here is an example showing what kind of error would be like without this fix
> <img width="543" alt="Screen Shot 2022-11-03 at 08 53 46" src="https://user-images.githubusercontent.com/11330831/199770007-cd6669dc-1b27-4505-81f5-9fc97ec84108.png">


## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
